README.md

```diff
@@ -43,14 +43,14 @@ The ultimate service for managing multi-channel notifications with a single API.
     <a href="https://twitter.com/novuhq">Twitter</a>
   </p>
 
-## ⭐️ Why
+## ⭐️ Why Novu?
 
-Building a notification system is hard, at first, it seems like just sending an email, but in reality, it's just the beginning. Users today expect a multi-channel communication experience via email, SMS, push, chat, and other channels. An ever-growing list of providers is popping up daily, and notifications are spread around the code. Novu's goal is to simplify notifications and provide developers with the tools to create meaningful communication between the system and its users.
+Building a notification system is hard, at first, it seems like just sending an email, but in reality, it's just the beginning. Users today expect a multi-channel communication experience via email, SMS, push, chat, and other channels. An ever-growing list of providers is popping up daily, and notifications are spread around the world. Novu's goal is to simplify notifications and provide developers with the tools to create meaningful communication between a system and its users.
```

Comment:
This change _changes_ the message as Novu is targeted at developers and tech companies. Please roll it back.